lib/chef/knife/core/subcommand_loader.rb

```diff
@@ -28,8 +28,8 @@ class SubcommandLoader
       attr_reader :chef_config_dir
       attr_reader :env
 
-      def initialize(chef_config_dir, env=ENV)
```

Comment:
That was mostly likely just for dependency injection in the tests. Looking around GitHub, I don't see it being used outside of Chef a lot, just copied into some vendored Chef+Ruby as a git repo thing: https://github.com/search?l=ruby&q=%27Chef%3A%3AKnife%3A%3ASubcommandLoader%27&ref=searchresults&type=Code&utf8=✓ (filter out HTML results).